econml/_ortho_learner.py

```diff
@@ -592,9 +598,9 @@ def fit(self, Y, T, X=None, W=None, Z=None, *, sample_weight=None, sample_var=No
         """
         self._random_state = check_random_state(self.random_state)
         if check_input:
-            Y, T, X, W, Z, sample_weight, sample_var, groups = check_input_arrays(
-                Y, T, X, W, Z, sample_weight, sample_var, groups)
-            self._check_input_dims(Y, T, X, W, Z, sample_weight, sample_var, groups)
+            Y, T, X, W, Z, sample_weight, freq_weight, sample_var, groups = check_input_arrays(
```

Comment:
yeah but the current logic in `StatsmodelsLinearRegression` is either freq_weight or sample_var could be None, and we will replace with a vector of ones or zeros and throw a warning saying the inference will be invalid! But it's not a failure, ideally we expect users input these two args together, that's why I wrote the docstring this way, maybe we should remove that specification?

Reply:
Sure! I add that, then the check input warnings from `StatsmodelsLinearRegression` will never be called since it will throw an error first here. I am bit concerned that this restriction is too strong, although we do expect the two args are used together.